src/Clicksign/ClientBase.php

```diff
@@ -79,6 +85,7 @@ protected function doRequest($url, $method, $data, $contentType = null)
         curl_setopt($c, CURLOPT_URL, $url);
         curl_setopt($c, CURLOPT_SSL_VERIFYHOST, 0);
         curl_setopt($c, CURLOPT_SSL_VERIFYPEER, 0);
+        curl_setopt($c, CURLOPT_PROXY, $this->proxy);
```

Comment:
Se `$this->proxy` for **NULL**, `CURLOPT_PROXY` é ignorado realizando a requisição sem o uso de PROXY.

`CURLOPT_PROXY` está disponível desde o PHP 4.0.2